composer.json

```diff
@@ -50,7 +50,8 @@
         "domnikl/statsd": "3.0.*",
         "influxdb/influxdb-php": "1.15.*",
         "bacon/bacon-qr-code": "2.0.2",
-        "phpmailer/phpmailer": "6.1.7"
+        "phpmailer/phpmailer": "6.1.7",
+        "chillerlan/php-qrcode": "^4.1"
```

Comment:
4.2 seem to be the latest version.

Reply:
Yes, forgot to apply the change on the VM env, sorry